website_snippet_carousel_product/static/src/js/snippet.options.js

```diff
@@ -113,6 +113,9 @@ odoo.define("website_snippet_carousel_product.snippet_options", function (requir
                 var sdomain = domain || '';
                 self.$target.attr("data-domain", sdomain.replace(/'/g, '"'));
                 self._refreshAnimations();
+                self.__click = true;
+                self._select(false, self.$target);
+                self.$target.trigger('snippet-option-change', [self]);
```

Comment:
:thinking: I'm not saying this code doesn't work... But I understand nothing :grin: 

Could you please explain a bit what was happening here or put a couple of screenshots so we know what to review please? Sorry for being so dumb :nerd_face: